### PR TITLE
feat(ssh_tunnel): Delete command & exceptions

### PR DESCRIPTION
**Pull request by Antonio-RiveroMartnez**
### SUMMARY
- Adding the new DELETE command and its exception to our SSH Tunnel

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset/databases/commands/delete_ssh_tunnel.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+logger = logging.getLogger(__name__)
+
+
+class DeleteSSHTunnelCommand(BaseCommand):
+    def __init__(self, model_id: int):
+        self._model_id = model_id
+        self._model: Optional[SSHTunnel] = None
+
+    def run(self) -> Model:
```
Comment (hughhhh) [inline superset/databases/commands/delete_ssh_tunnel.py:39]: can we write a small unit test for deleting this the same way as the last PR we have

File: superset/databases/commands/exceptions.py [hunk 2]
```diff
@@ -176,3 +176,11 @@ class DatabaseOfflineError(SupersetErrorException):
 
 class InvalidParametersError(SupersetErrorsException):
     status = 422
+
+
+class SSHTunnelDeleteFailedError(DeleteFailedError):
+    message = _("SSH Tunnel could not be deleted.")
```
Comment (hughhhh) [inline superset/databases/commands/exceptions.py:182]: can you add a status `500` on this one

File: superset/databases/commands/delete_ssh_tunnel.py (hunk 1 above)
Comment (Antonio-RiveroMartnez) [reply superset/databases/commands/delete_ssh_tunnel.py:39]: Oh yeah! Let me add those for this and the update one

File: superset/databases/commands/exceptions.py (hunk 2 above)
Comment (Antonio-RiveroMartnez) [reply superset/databases/commands/exceptions.py:182]: The parent class already has `status = 500` so it should be already there.

File: superset/databases/commands/exceptions.py [hunk 3]
```diff
@@ -176,3 +176,11 @@ class DatabaseOfflineError(SupersetErrorException):
...
+
+
+class SSHTunnelNotFoundError(CommandException):
```
[the remark this answers is not in the document]
Comment (Antonio-RiveroMartnez) [reply superset/databases/commands/exceptions.py:185]: Will do 👍

**Review by hughhhh — APPROVED** [review]

**Pull request merged**